pkg/awsv1alpha1/accounts.go

```diff
@@ -0,0 +1,29 @@
+package awsv1alpha1
+
+import (
+	"context"
+	"fmt"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetAWSAccount(ctx context.Context, cli client.Client) ([]string, error) {
```

Comment:
This should be more specific
`GetAccountCRsToReset` or something similar. Right now this reads like getting every AWS account, not getting AWS IDs from Account CRs